cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/test/java/io/cdap/cdap/datapipeline/AutoJoinerTest.java

```diff
@@ -169,6 +170,8 @@ public void testCaseSensitivity() throws Exception {
     AppRequest<ETLBatchConfig> appRequest = new AppRequest<>(APP_ARTIFACT, config);
     ApplicationId appId = NamespaceId.DEFAULT.app(UUID.randomUUID().toString());
     ApplicationManager appManager = deployApplication(appId, appRequest);
+    String version = getAppDetail(appId).getAppVersion();
+    appId = new ApplicationId(appId.getNamespace(), appId.getApplication(), version);
```

Comment:
Why do you need this? I don't see appId used below

Reply:
removed it